docs/src/examples.md

```diff
@@ -155,3 +155,290 @@ FrankWolfe.plot_results(
 )
 plot!(size=(3000, 2000),legendfontsize=30)
 ```
+
+## Polynomial Regression
+
+The following example features the LMO for the ``\ell_1`` norm ball. Given input/output pairs ``\{x_i,y_i\}_{i=1}^N`` and sparse coefficients ``c_j``, where
```

Comment:
the highlight is more the polynomial regression here more than the L1 ball

Reply:
better?